rm.Extensions/DateTimeExtension.cs

```diff
@@ -50,5 +51,36 @@ public static DateTime AsUtcKind(this DateTime datetime)
 		{
 			return DateTime.SpecifyKind(datetime, DateTimeKind.Utc);
 		}
+
+		/// <summary>
+		/// Returns the date in year month ordinal format
+		/// </summary>
+		public static string ToYearMonthOrdinal(this DateTime date)
```

Comment:
The name of the method 'DateTime.ToYearMonthOrdinal()' looks off to me since it's `DateTime.Day`'s ordinal.